.env

```diff
@@ -0,0 +1 @@
+GENERATE_SOURCEMAP=false
```

Comment:
will it have effect for local development?

Reply:
No it only applies to production. You can check this link https://create-react-app.dev/docs/advanced-configuration/